jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/Dumpable.java

```diff
@@ -50,24 +56,56 @@ default String dump()
     void dump(Appendable out, String indent) throws IOException;
 
     /**
-     * Utility method to implement {@link #dump()} by calling {@link #dump(Appendable, String)}
+     * Utility method to call dump to a {@link String}
      *
      * @param dumpable The dumpable to dump
      * @return The dumped string
+     * @see #dump(Appendable, String)
      */
     static String dump(Dumpable dumpable)
     {
         StringBuilder b = new StringBuilder();
+        dump(dumpable, b);
+        return b.toString();
+    }
+
+    DateTimeFormatter UTC_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").withZone(ZoneOffset.UTC);
+    DateTimeFormatter LOCAL_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS Z").withZone(ZoneId.of(System.getProperty("user.timezone")));
```

Comment:
These are now public constants that are just an implementation detail.

I would just allocate them on-the-fly for the formatting, if necessary.

Actually, I think there are predefined constants in `DateTimeFormatter`, but I would not even do that.
Just calling `Instant.toString()` formats it in UTC following the standard.

Reply:
How about this output: 
```
key: +- bean, += managed, +~ unmanaged, +? auto, +: iterable, +] array, +@ map, +> undefined
JVM: Oracle Corporation OpenJDK 64-Bit Server VM 22+36-2370; OS: Linux amd64 6.5.0-35-generic; Jetty: 12.0.10-SNAPSHOT; CPUs: 20; mem(free/total/max): 4,022/4,096/6,144 MiB
UTC: 2024-05-27T21:59:02.015802402Z; Europe/Rome: 2024-05-27T23:59:02.015802402+02:00
```